### PR TITLE
Add search for token list, fixes #69

### LINKED ISSUE #69: Search for tokens
The token-list is already very long (and gets longer very fast - see https://github.com/MyEtherWallet/ethereum-lists/issues/191 ) - so we need a search function for tokens

### PR DESCRIPTION
**Pull request by friedger**
Search field has been added to select token activity.

screenshot_1509898257

### CODE REVIEW

File: app/src/main/java/org/walleth/activities/SelectTokenActivity.kt [hunk 1]
```diff
@@ -56,6 +57,20 @@ class SelectTokenActivity : AppCompatActivity() {
         fab.setOnClickListener {
             startActivityFromClass(CreateTokenDefinitionActivity::class)
         }
+        search.setOnQueryTextListener(object : SearchView.OnQueryTextListener {
+            override fun onQueryTextSubmit(query: String?): Boolean {
```
Comment (ligi) [inline app/src/main/java/org/walleth/activities/SelectTokenActivity.kt]: nit: like `override fun onQueryTextSubmit(query: String?) = true ` a bit more and it is the style used in WALLETH - or are you opposed to this?

File: app/src/main/java/org/walleth/activities/SelectTokenActivity.kt [hunk 2]
```diff
@@ -56,6 +57,20 @@ class SelectTokenActivity : AppCompatActivity() {
...
+                return true;
+            }
+
+            override fun onQueryTextChange(newText: String?): Boolean {
+                if (newText != null) {
+                    tokenListAdapter.filter(newText)
+                }
+                return true;
```
Comment (ligi) [inline app/src/main/java/org/walleth/activities/SelectTokenActivity.kt]: nit: please no semicolons

**Comment by ligi** [comment]
Just tried it out - there seems to be a problem when there was not yet a query entered - for me the list is then empty.
screenshot_1509938887

**Comment by ligi** [comment]
Thinking about it a bit more - What do think about a search in the ActionBar as I do in SurvivalManual

screenshot_1509939892

https://github.com/ligi/SurvivalManual/blob/a5136855844a53d4cdc8ea058dbfa3774e133505/android/src/main/kotlin/org/ligi/survivalmanual/ui/MainActivity.kt#L133

**Comment by friedger** [comment]
Search in the action bar feel more like one feature among other, not like being the main action. Not sure what you would like to have. However, the text field takes a lot of UI space.

**Pull request closed without merging**